packages/risc0/script/config.toml

```diff
@@ -15,6 +15,6 @@ inputValidatorAddress = "0x0000000000000000000000000000000000000000"
 # You can add additional profiles here
 [profile.custom]
 chainId = 31337
-riscZeroVerifierAddress = "0x0000000000000000000000000000000000000000"
+riscZeroVerifierAddress = "0x0165878A594ca255338adfa4d48449f69242Eb8F" # Based on default deployment address using local anvil node
```

Comment:
This address should remain `address(0)` since the [[deployment script](https://github.com/gnosisguild/CRISP/blob/main/packages/risc0/script/Deploy.s.sol#L97-L105)](https://github.com/gnosisguild/CRISP/blob/main/packages/risc0/script/Deploy.s.sol#L97-L105) automatically deploys this contract.  
Otherwise, the first time the user runs the deployment script, this contract won’t exist on the local node because the script would skip it.

Reply:
ohh I assumed this must need to be validated as well. Therefore, the user do not have to set it manually in the environment. Got it!